index.bs

```diff
@@ -1074,6 +1087,88 @@ spec: webidl
     </p>
   </section>
 </section>
+<section>
+  <h2 id="automation">
+    Automation
+  </h2>
+  <p>
+    For the purposes of user-agent automation and application testing, this
+    document defines the following <a>extension command</a> for the
+    [[WebDriver]] specification.
+  </p>
+
+  <section>
+    <h3 id="set-permission-command">
+      Set Permission
+    </h3>
+    <table>
+      <tbody>
+        <tr>
+          <th>HTTP Method</th>
+          <th><a lt="extension command prefix">Prefix</a></th>
+          <th><a lt="extension command name">Name</a></th>
+        </tr>
+        <tr>
+          <td>POST</td>
+          <td>permissions</td>
```

Comment:
Maybe I am not understanding correctly, but in the specification is it not sufficient to just have the "permissions" as the URI.
That way in WebDriver implementations they can be /session/{session id}/vendorspecificuri/permissions

Reply:
@JKereliuk It's actually the consumer's responsibility to define that part of the URL. [That wasn't clear when I wrote this patch back in June](https://github.com/w3c/webdriver/issues/961), and it's only since then that we've made the interaction explicit (see https://github.com/w3c/webdriver/pull/956 and https://github.com/w3c/webdriver/pull/962). So Jeffrey's request here makes sense to me; the current text is a relic of a issue we've since resolved in WebDriver.